src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,19 @@
+package ladder.domain;
+
+import java.util.List;
+
+public class Line {
+	List<Boolean> points;
+
+	public Line(int countOfPerson) {
+		this(new RandomLineStrategy(), countOfPerson);
+	}
+
+	public Line(LineStrategy lineStrategy, int countOfPerson) {
+		this.points = lineStrategy.points(countOfPerson);
+	}
+
+	public List<Boolean> getLine() {
+		return points;
+	}
```

Comment:
method의 네이밍은 getLine 인데 points 를 반환하고 있네요! 😄 

```suggestion
	public List<Boolean> getPoints() {
		return this.points;
	}
```